resources/asciidoctor/lib/edit_me/extension.rb

```diff
@@ -74,8 +74,16 @@ def link_for(block)
       url = edit_url block
       return '' unless url
 
+      if block.document.attr 'private_edit_urls'
+        css_class = 'edit_me edit_me_private'
+        title = 'Editing on GitHub is available to Elastic'
```

Comment:
I updated the text for the tooltip after we talked; given in how many places we show the edit icon, I wanted to make the message more positive (i.e. just say who CAN edit rather than that you need to be an authorized user).